application-analytics-ui/src/main/resources/Analytics/Code/Configuration.xml

```diff
@@ -90,28 +104,14 @@
         <name>trackingCode</name>
         <number>1</number>
         <picker>1</picker>
-        <prettyName>Matomo Tracking Code</prettyName>
```

Comment:
This does not seem just related to the export, but rather a change. Was it the wrong value on master? Just `Tracking code` seems better indeed